README.md

```diff
@@ -20,8 +20,8 @@
     <li>User accounts - Doing this allows for many more features like tracking reaction speed over a longer period of time.</li>
 </ul>
 
-<h2><u>The Algorithm Used to get Reaction Speed</u></h2>
-<p>Your score should be accurate because the algorithm is that every time a new color is generated, a start date object is created, and when you click the board, an end date object is made, and then I simply get the difference between the two dates and display it in milliseconds. This works much better than relying on setInterval/setTimeout/requestAnimationFrame as those methods are not 100% consistent.</p>
+<h2><u>The Algorithm Used to get Reaction Speed</u></h2>
+<p>Your score should be accurate because the algorithm is that every time a new color is generated, a start time is recorded using the browser's <code>performance.now()</code> API, and when you click the board we capture another timestamp. The difference between these high&#x2011;resolution times is displayed in milliseconds, providing more precise results than the older <code>Date</code> approach.</p>
```

Comment:
The purpose of  this paragraph is to explain to the user how the score is measured. The user does not need to know about how the score was previously calculated. Please change the copy to reflect that.